src/locales/de-DE.json

```
@@ -182,6 +185,15 @@
           "required": "Die KoPers.-Nr. ist erforderlich."
         }
       },
+      "title": {
+        "ERROR_LOADING_USER": "Fehler beim Laden des Benutzers.",
+        "PERSONALNUMMER_NICHT_EINDEUTIG": "KoPers.-Nr. nicht eindeutig",
```

All other messages and with a period.
```suggestion
        "PERSONALNUMMER_NICHT_EINDEUTIG": "KoPers.-Nr. nicht eindeutig.",
```